packages/eas-cli/src/utils/statuspageService.ts

```diff
@@ -26,6 +26,7 @@ const humanReadableServiceName: Record<StatuspageServiceName, string> = {
   [StatuspageServiceName.EasUpdate]: 'EAS Update',
   [StatuspageServiceName.GithubApiRequests]: 'GitHub API Requests',
   [StatuspageServiceName.GithubWebhooks]: 'Github Webhooks',
+  [StatuspageServiceName.EasWorkflows]: 'EAS Workflows',
```

Comment:
Added this because after updating the gql code with `yarn generate-graphql-code` the typecheck for this failed because a new enum has been added.